### ISSUE TITLE
Incorrect Variable Passed Turned into String?

### ISSUE THREAD

**Issue by winrid** [adamghill/django-unicorn#630]
Basically, it seems like child components cannot distinguish between a valid passed in variable vs an invalid variable/gibberish.

Just ran into this. If I do this:

```
{% unicorn 'event_bulleted_summary' user=this_variable_does_not_exist event=event %}
```

I get this error:

```
The component class 'sidewaysdata.components.event_bulleted_summary.EventBulletedSummaryView' could not be loaded: 'str' object has no attribute 'is_anonymous'
```

Because:

```
profile = self.user.profile if self.user and not self.user.is_anonymous else None
```

User is "truthy" - but it's a string! :D

Maybe `user` should become None here?

It can be None in the component:

```
user: Optional[User] = None
```

I feel like this is something folks will run into a lot and maybe components should fail fast with an error somehow. I am not sure how you'd detect that yet with Django's templating system.

**Comment by adamghill** [comment]
Does this still happen in 0.58.0? I might be able to handle this case if it's still a problem.

**Comment by winrid** [comment]
@adamghill Yep:

```
$ ls -lah venv/lib/python3.10/site-packages/ | grep django_unicorn
drwxrwxr-x   9 winrid winrid 4.0K Dec 24 00:59 django_unicorn
drwxrwxr-x   2 winrid winrid 4.0K Dec 24 00:59 django_unicorn-0.58.0.dist-info
```
In `sidewaysdata/templates/unicorn/find_events.html`

change

```
{% unicorn 'event_bulleted_summary' user=user event=event %}
```

to

```
{% unicorn 'event_bulleted_summary' user=abcdef event=event %}
```

error is:

```
The component class 'sidewaysdata.components.event_bulleted_summary.EventBulletedSummaryView' could not be loaded: 'str' object has no attribute 'is_anonymous'
```

because `user` becomes a string "abcdef".

**Comment by adamghill** [comment]
PR is merged so it will get fixed in 0.60.0. Closing.
